### ISSUE TITLE
AMPMessage rfcData not working

### ISSUE THREAD

**Issue by joeherold** [Airmail/AirmailPlugIn-Framework#18]
Wehen creating a plugin in the -(void)sendToInbox: (NSMenuItem *)item{} the message object does not have a rfcData output.

The property **rfcData** only returns **nil** and **not NSData** until a mail has **manually been moved/copied to the desktop** for examle. This is a pain.

![log](https://cloud.githubusercontent.com/assets/3911545/8065513/32423344-0ee2-11e5-8409-e369507a371a.png)

I wrote a plugin for Podio.com for creating tasks emails an collaborate with it. There i want to upload an *.eml file aswell. so in neeeeed the [msg rfcData]

How can I achieve this? Did you subset your classes from libmailcore?

![nil](https://cloud.githubusercontent.com/assets/3911545/8065423/a4778c1c-0ee1-11e5-97e3-85b7c8d0e119.png)

**Comment by KodaKoder** [comment]
msg.rfcData not working?

**Comment by joeherold** [comment]
Yes. As mentioned above. It returns only nil before a message has been manually dragged outside airmail.
